backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHParquetSuite.scala

```diff
@@ -2253,5 +2253,13 @@ class GlutenClickHouseTPCHParquetSuite extends GlutenClickHouseTPCHAbstractSuite
     }
   }
 
+  test("GLUTEN-4115 aggregate without any function") {
+    val sql =
+      """
+        | select n_regionkey, n_nationkey from nation group by n_regionkey, n_nationkey
```

Comment:
For case like this, can we just treat it as complete stage? If so, distinct outputs can be got from partial stage.

Reply:
It will cause performance issue